appveyor.yml

```diff
@@ -7,20 +7,30 @@ environment:
     PYTHONUNBUFFERED: 1
 
   matrix:
+    - PYTHON: "C:\\Miniconda36-x64"
+      CONDA_PY: "27"
+      ARCH: "64"
     - PYTHON: "C:\\Miniconda36-x64"
       CONDA_PY: "35"
       ARCH: "64"
+    - PYTHON: "C:\\Miniconda36-x64"
+      CONDA_PY: "36"
+      ARCH: "64"
 
 install:
   - set PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
+  - conda config --set always_yes yes --set changeps1 no
```

Comment:
Is `--set changeps1 no` necessary?

Reply:
Nope, its an older artifact I think from where I copied this from, I'll change that

Reply:
Actually its to match the travis stuff